packages/react-router/src/not-found.tsx

```diff
@@ -4,7 +4,7 @@ import { useRouterState } from './useRouterState'
 import { RegisteredRouter, RouteIds } from '.'
 
 export type NotFoundError = {
-  global?: boolean
+  _global?: boolean
```

Comment:
Is there a reason for this api surface to change?
This'll be a breaking change for anyone currently calling `notFound({ global: true })` yes?

Reply:
Hmmm yeah. But I thought I left in a backwards compatibility layer…

Reply:
Not sure if that'd help this though, as this is a user input type.
Since the type `NotFoundError` is the same user input used when throwing a `notFound()`, changing it makes it so now you'd have to `throw notFound({ _global:true })` instead of just `throw notFound({ global: true })`.

Reply:
I’ll fix this or at the very least support both officially with a deprecation notice.